spec/system/members/cash_register_proficiency_spec.rb

```diff
@@ -6,7 +6,10 @@
   let(:mission) { create(:mission) }
   let(:jack) { create(:member, cash_register_proficiency: :proficient) }
 
-  before { sign_in jack }
+  before do
```

Comment:
gray zone: you shadow the driver config. you can instead make a config driver for each context putting the driver config in the before L15